portal-kernel/src/com/liferay/portal/kernel/util/SystemProperties.java

```diff
@@ -60,6 +60,10 @@ public static String get(String key) {
 		return value;
 	}
 
+	public static String[] getArray(String key) {
+		return _arrayValues.computeIfAbsent(key, k -> StringUtil.split(get(k)));
+	}
```

Comment:
@W2jx @dantewang As we are parsing reference at runtime to get the updated value, adding a cache here actually breaks this. I know we have the same cache in PropsUtil so you may not bring any regression here. But the logic itself does not make sense.

Also, we need add test for this.

Tina.